codalab/apps/web/static/js/vendor/jquery-editable-poshytip.js

```diff
@@ -2187,7 +2186,7 @@ Makes editable any HTML element on the page. Applied as jQuery method.
         @type string
         @default 'Empty'
         **/         
-        emptytext: 'Empty',
+        emptytext: '{empty}',
```

Comment:
We can make it none but I am unable to make it <none> (with the <>) because they are interpreted as tags in xml which is used by react.